### ISSUE TITLE
Feature Request: Add line number to console output.

### ISSUE THREAD

**Issue by flukeout** [mozilla/thimble.mozilla.org#2445]
One of our Teacher users would love to use the built-in console, but without seeing a line number feels that it is not useful enough.

**Comment by apoorvasingh17** [comment]
Hi, I'd like to work on this. Can someone give me any pointers?

**Comment by aryanshar** [comment]
Hey, i think i can fix this issue. Please assign me this.Thanks

**Comment by ryanwarsaw** [comment]
@apoorvasingh17 I've assigned this ticket to you, I'm not familiar with the console feature but feel free to get your Development environment setup, and join our Mattermost if you have questions!

@aryanshar Since @apoorvasingh17 asked to be assigned to this issue first we have to oblige and give them the opportunity to work on it, but you're more than welcome to find another issue to work on!

**Comment by apoorvasingh17** [comment]
I think this is a bit tough for me as I have just started contributing. You can assign this to @aryanshar . Sorry for the inconvenience.

**Comment by ryanwarsaw** [comment]
@apoorvasingh17 Not a problem. Contributing can be daunting at first, but it's incredibly rewarding. If you find another issue you'd like to work on, just let us know. I can also help you find an easier issue if that's something you're interested in. Also be sure to join our Mattermost, we're a very friendly group (also filled with fellow contributors) and we're happy to help you get started, and answer any questions you have. Alternatively, you can also ask them here! :-D

@aryanshar Did you still have an interest in tackling this issue?

**Comment by aryanshar** [comment]
@ryanwarsaw Yes, i would like to work on this issue. But a little help in how should i begin with is required, as i am new to open source.

**Comment by ryanwarsaw** [comment]
@aryanshar A good place to start would be to get your environment setup, and read through the getting started documentation in the README.md and CONTRIBUTING.md files respectively. 

If you have any specific questions after that just let me know.

**Comment by jmdjaini** [comment]
@ryanwarsaw  hello sir , i am new here to github and want to contribute . i am good in javascript , could you help me ??

**Comment by ryanwarsaw** [comment]
@jmdjaini Try to skim down this list of "Good First Bugs", and if you find one that interests you and isn't already being worked on, just leave a comment and we can go from there.

https://github.com/mozilla/thimble.mozilla.org/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Bug%22

**Comment by ajlomagno** [comment]
@flukeout I'm a student at Seneca and I wanted to work on this for my open source class. I'd like to give it a go if no one else is currently doing so.

**Comment by flukeout** [comment]
Hey @ajlomagno - sure thing yeah give it a shot! Let us know if you have questions. It looks like work by other potential contributors has stalled out here.

**Comment by ajlomagno** [comment]
I'll be sure to stop by and ask any questions if I need some pointers. Thanks @flukeout!

**Comment by gideonthomas** [comment]
@ajlomagno, one thing that might help...this issue needs to be fixed in brackets...so I don't think you need to setup Thimble, and should be able to test this out with Brackets itself. Instructions for setting up Brackets locally can be found in the readme in the [Brackets repo](https://github.com/mozilla/brackets)

**Comment by Avedis777** [comment]
I'm a student at Seneca college learning open source, and I was hoping to work on this bug for my course.  If no one else is currently working on it, I'd like to give it a try.

**Comment by ajlomagno** [comment]
@gideonthomas, that'll definitely help give me a good place start. Thanks a lot!

**Comment by gideonthomas** [comment]
@Avedis777 looks like @ajlomagno is working on it. You might find an issue here: https://github.com/mozilla/thimble.mozilla.org/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20-label%3A%22Assigned%20to%20Contributor%22 that might interest you

**Comment by ajlomagno** [comment]
Hey, @flukeout I've made some great progress on getting the console line numbers added but as @gideonthomas mentioned above this is an addition that's primarily focused in [bramble (brackets)](https://github.com/mozilla/brackets). When it's time to make a pull request should I create one for this repository or for bramble's repository?

**Comment by gideonthomas** [comment]
Since your code changes touches the brackets repository, make your PR against that.

**Comment by flukeout** [comment]
Looking forward to the PR @ajlomagno!
